docs/docs/quickstart.md

```diff
@@ -2,21 +2,24 @@
 
 ## Dependencies
 
-http4s is available for Scala 2.12, 2.13, and 3.
-
-You can add http4s to your build by adding its modules to your dependencies in `build.sbt`.
-Here are some common dependencies, more are available on the [integrations page][integrations].
+http4s is available for Scala 2.12, 2.13, and 3.2+. You can add http4s to your build by adding its modules to `libraryDependencies` in your `build.sbt`.
 
 ```scala
 val http4sVersion = "@VERSION@"
 
 libraryDependencies ++= Seq(
-  // get up and running with a client and server
   "org.http4s" %% "http4s-ember-client" % http4sVersion,
   "org.http4s" %% "http4s-ember-server" % http4sVersion,
   "org.http4s" %% "http4s-dsl"          % http4sVersion,
+)
+```
+
+This brings in the [Ember HTTP client and server implementation][ember] as well as the the [routing DSL][dsl]. This is _most likely_ what you want. Other [integration modules][integrations] are available as well, such as for working with [JSON], or using alternative client/server implementations, such as [Netty].
 
-  // for core classes and traits, e.g. `Client[F]`
+Depending on your usecase, you may want to consider one of several other http4s modules. For example, if you are writing a library, you probably want to develop it against the generic http4s interfaces, without hard-coding a particular implementation, such as Ember.
```

Comment:
It might also be worth mentioning that some of these others may be versioned separately.